llorens/day01/Day01Solver.py

```diff
@@ -1,22 +1,11 @@
-import AoCReader
+def solve_part1(sequence):
 
-if __name__ == "__main__":
+    return sum(int(sequence[i]) for i in range(len(sequence)) if sequence[i] == sequence[(i+1) % len(sequence)])
 
-    sequence = AoCReader.get_input("01")
 
-    part1 = 0
-    part2 = 0
+def solve_part2(sequence):
 
-    size = len(sequence)
-    displacement = size/2
+    return sum(int(sequence[i]) for i in range(len(sequence)) if sequence[i] == sequence[(i+int(len(sequence)/2)) %\
+                                                                                         len(sequence)])
```

Comment:
Things get a little hackier here, but it's just my opinion. Good job 👍

Reply:
Yes I think this line could be more descriptive, but he wants to do all in one line, this is not a elegant line, but he did it.

One day like a mathematician writing a elegant equation, he writes a elegant line resolving a big problem, (or destroying the entire world)